tls/extensions/s2n_client_supported_groups.c

```diff
@@ -78,6 +78,25 @@ static int s2n_client_supported_groups_send(struct s2n_connection *conn, struct
     return S2N_SUCCESS;
 }
 
+S2N_RESULT s2n_client_supported_groups_parse_count(struct s2n_stuffer *extension, uint16_t *count)
```

Comment:
Nit: This name sounds kind of nonsensical to me. What about s2n_supported_groups_count()? I don't think it _needs_ to have "client" prefaced on it.

Reply:
I removed "client" but kept "parse" to make it clear it was reading the count from the stuffer.